client/curriculum/html/AboutHtml.tsx

```diff
@@ -0,0 +1,335 @@
+            <strong className="formatted">{'<section>'}</strong>
+            {' '}
+            are drept continut alte doua elemente (
+            <strong className="formatted">{'<h1>'}</strong>
+            {' si '}
+            <strong className="formatted">{'<p>'}</strong>
+            ), in timp ce aceste doua elemente au in interior continut
+            de tip text.
+          </p>
+          <LessonTip>
+            Exista si elemente care nu au nevoie de continut (de exemplu,
+            {' elementul '}
+            <strong className="formatted">input</strong>
+            ). Acestea se numesc
+            {' '}
+            <strong>self-closing tags</strong>
+            , iar modul de scriere este putin diferit, si anume
+            {' '}
+            <strong className="formatted">{'<element />'}</strong>
+            .
+          </LessonTip>
+        </section>
+        <section>
+          <LessonHeading as="h3" id="atribute">
+            Atribute
+          </LessonHeading>
+          <p>
+            Fiecare element poate avea o serie de atribute prin care noi putem fie
+            sa controlam comportamentul acestuia, fie sa il identificam ulterior in
+            CSS sau JS. Atributele sunt perechi de tip
+            {' '}
+            <strong className="formatted">cheie=“valoare”</strong>
+            {' '}
+            care sunt scrise in interiorul unui opening tag.
+          </p>
+          <p>
+            Revenind la analogia de la inceput, daca ne gandim la un bec, atribute
+            posibile ar putea fi tipul becului (LED sau incandescent), culoarea becului,
+            intensitatea acestuia, marimea, etc.
+          </p>
+          <p>
+            In HTML, un element bec care este de
+            {' '}
+            <strong>tip LED</strong>
+            , de
+            {' '}
+            <strong>culoare albastra</strong>
+            , ar putea fi scris dupa cum urmeaza:
+          </p>
+          <Highlight
+            className="my-5"
+            language="html"
+            code={`
+<bec tip=“LED” culoare="albastru” />
+`}
+          />
+          <LessonTip>
+            Becul nu este un element valid in HTML. Exemplul de mai sus a fost
+            scris doar pentru a continua analogia inceputa in primele paragrafe.
+          </LessonTip>
+
+          <p>
+            Hai sa vedem si cum putem folosi doua elemente existente si foarte intalnite:
+            {' '}
+            <strong>titlul</strong>
+            {' si '}
+            <strong>paragraful</strong>
+            .
+          </p>
+          <Highlight
+            className="my-5"
+            language="html"
+            code={`
+<h1>Acesta este un titlu</h1>
+<p>Acesta este un paragraf</p>
+`}
+          />
+          <p>Acum, vom adauga si cateva atribute pe aceste elemente:</p>
+          <Highlight
+            className="my-5"
+            language="html"
+            code={`
+<h1 id=“main-title”>Acesta este un titlu</h1>
```

Comment:
I think we should change what attributes we demo, and use things that actually make a difference directly.

`id` and `class` by themselves don't do anything, so I worry it might be hard for people that may run the demos on their machine to understand what is going on.